GiniVision/Classes/ReviewContainerViewController.swift

```diff
@@ -78,10 +77,35 @@ internal class ReviewContainerViewController: UIViewController, ContainerViewCon
         displayContent(contentController)
     }
     
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        
+        // Configure back button. Needs to be done here because otherwise the navigation controller would be nil
+        guard let _ = navigationItem.leftBarButtonItem else {
```

Comment:
I'm not sure about using `guard` here. As far as I can tell we are not enforcing any preconditions here, there are no statements after the `guard`. We check the existence of the bar button item and if it's missing we add it.

I propose doing it the "old school" way, which conveys the intent on first sight:

```
if navigationItem.leftBarButtonItem == nil {
    setupLeftNavigationItem(usingResources: backButtonPreferredResource(), selector:#selector(back))
}
```

Reply:
Yes you are right. Too much Swift in my head... 😅